src/auto-delegates/extensions/AutoDelegateOpenZeppelinGovernor.sol

```diff
@@ -43,7 +43,8 @@ abstract contract AutoDelegateOpenZeppelinGovernor {
     returns (uint256 _proposalDeadline, uint256 _forVotes, uint256 _againstVotes, uint256 _quorumVotes)
   {
     _proposalDeadline = IGovernor(_governor).proposalDeadline(_proposalId);
+    uint256 _proposalSnapshot = IGovernor(_governor).proposalSnapshot(_proposalId);
```

Comment:
I considered adding a test, but it seemed like overkill as we currently are mocking a static quorum. Happy to add if it isn't.

Reply:
#79